### PR TITLE
Deploy dev to main

### PR DESCRIPTION
**Pull request by phc979**

### CODE REVIEW

**Review by linkpond — COMMENTED** [review]
I agree this coding

File: src/main/java/com/team5/fandom/repository/ArtistRepository.java [hunk 1]
```diff
@@ -5,7 +5,9 @@
 
 import com.team5.fandom.entity.Artist;
 
-@Repository
-public interface ArtistRepository extends JpaRepository<Artist,Integer> { 
+import java.util.List;
```
Comment (phc979) [inline src/main/java/com/team5/fandom/repository/ArtistRepository.java:8]: GOOOD

File: src/main/java/com/team5/fandom/repository/PostRepository.java
Comment (syjoy1993) [inline src/main/java/com/team5/fandom/repository/PostRepository.java:1]: ok

**Review by linkpond — APPROVED** [review]
I agree

**Review by syjoy1993 — COMMENTED** [review]
ok

**Review by syjoy1993 — APPROVED** [review]

**Pull request merged**